texastribune/donations
# Pre-UMP front-end stuff

#### What's this PR do?
+ Upgrades to Babel 7
+ Adds `async/await` capabilities
+ Updates Autoprefixer settings
+ Updates `vue-loader`
+ Adds the ability to use environment variables in Webpack builds (for Auth0 stuff)

#### Why are we doing this? How does it help us?
Gearin' up for UMP.

#### How should this be manually tested?
Try a few test donations on `/donate` and `/circle`. Ensure they show up in #stripe.

#### How should this change be communicated to end users?
NA.

#### Are there any smells or added technical debt to note?
Less debt, but I do have a question for @x110dc: How do I add config vars in Heroku that will carry over to all PR branches? I'd like to add the Auth0 test-tenant values.

#### What are the relevant tickets?
https://3.basecamp.com/3098728/buckets/10357222/todos/1720704325

#### Have you done the following, if applicable:
